scripts/GasDataToCsv.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
```

Comment:
Is this file (and the generated csvs) used by the reporter or something?

Reply:
it is not, I can remove this file as well as data/ but I would probably use it next time I come back to gas analysis.